samples/Ecs/AudioEcsSample/main.cpp

```diff
@@ -0,0 +1,230 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
+// for more information.
+
+#include <NovelRT.h>
+#include <time.h>
+
+using namespace NovelRT::Ecs::Audio;
+
+int main(int /*argc*/, char* /*argv*/[])
```

Comment:
You could probably just delete the parameters here. I don't think main requires you have them, esp. if you're not using them?

Reply:
Done